vllm/config.py

```diff
@@ -3150,7 +3149,7 @@ def model_post_init(self, __context: Any) -> None:
         #    and it is not yet a priority. RFC here:
         #    https://github.com/vllm-project/vllm/issues/14703
 
-        if Version(torch.__version__) >= Version("2.6"):
+        if Version(importlib.metadata.version('torch')) >= Version("2.6"):
```

Comment:
Wondering why torch.__version__ doesn't work here?

Reply:
@aarnphm's suggestion - docs build was failing for some reason and I figured it should be consistent. Should I revert this?

Reply:
Do we install torch on buildthedocs? If that is the case then `torch.__version__` should work?

but also https://github.com/pytorch/pytorch/blob/main/torch/torch_version.py...., seems like a massive hack (https://github.com/pytorch/pytorch/blob/main/torch/version.py.tpl)

`metadata.version` should be the most stable.